install/roles/system-prepare/tasks/main.yml

```diff
@@ -293,3 +293,15 @@
   template:
     src: bashrc
     dest: /root/.bashrc
+
+- name: Create the folder for renewal hooks
+  file:
+    path: /etc/letsencrypt/renewal-hooks/deploy
+    state: directory
+
+- name: Use python3 interpreter by default
+  file:
+    src: /usr/bin/python3
+    dest: /usr/bin/python
+    state: link
+    force: yes
```

Comment:
There seems to be a package for that.
https://unix.stackexchange.com/questions/609855/how-to-make-python-an-alias-of-python3-systemwide-on-debian
https://www.debian.org/doc/packaging-manuals/python-policy/index.html

With more details from the packaging doc, maybe it would be more robust just
to change shebang line in scripts to `python3`.